app/src/main/java/com/twilio/video/app/util/CameraCapturerCompat.kt

```diff
@@ -0,0 +1,85 @@
+package com.twilio.video.app.util
+
+import android.content.Context
+import android.graphics.ImageFormat
+import android.hardware.camera2.CameraCharacteristics
+import android.hardware.camera2.CameraManager
+import android.hardware.camera2.CameraMetadata
+import android.os.Build
+import com.twilio.video.Camera2Capturer
+import com.twilio.video.VideoCapturer
+import timber.log.Timber
+import tvi.webrtc.Camera1Enumerator
+import tvi.webrtc.Camera2Enumerator
+import tvi.webrtc.CameraEnumerator
+
+interface CameraCapturerCompat {
```

Comment:
I recommend borrowing the quickstartKotlin implementation of [CameraCapturerCompat](https://github.com/twilio/video-quickstart-android/blob/6.0.0-beta/quickstartKotlin/src/main/java/com/twilio/video/quickstart/kotlin/CameraCapturerCompat.kt).

I am fine with adding the `newInstance` method to create the object, but I don't think we need to break this class into separate classes.